### ISSUE TITLE
The stack hangs upon receiving a command on port 203 (Firmware Management Protocol)

### ISSUE THREAD

**Issue by Ramin-Mardani** [Lora-net/SWL2001#13]
https://github.com/Lora-net/SWL2001/blob/45e19e5bf783bf874b8c9825b76c4c5befa8bcba/smtc_modem_core/lorawan_packages/fuota_packages/lorawan_fmp_package.c#L489

The device will hang whenever I send a command on port 203 that is specified in Firmware Management Protocol Specification TS006-1.0.0. from the backend. For example, **DevVersionReq**. 

I will get the following information from the console:

```
lorawan_fmp_package_service_downlink_handler receive data on port 203
LoRaWAN App got an event
INFO: Event received: TXDONE
INFO: Transmission done
 lorawan_fmp_package launch REQUEST_REBOOT_TASK_MASK n lorawan_fmp_package launch REQUEST_REBOOT_TASK_MASK n lorawan_fmp_package launch REQUEST_REBOOT_TASK_MASK
```

The stack doesn't respond to the backend and the phrase `REQUEST_REBOOT_TASK_MASK n` will be endlessly repeated.

**Comment by MpmTwtg** [comment]
Having the same issue, 

Changing
ctx->fmp_task_ctx_mask |= REQUEST_REBOOT_TASK_MASK; 

to
ctx->fmp_task_ctx_mask |= ANS_CMD_TASK_MASK; 

Seems to be the fix,
The same problems occurs in fuota_packages_v2

**Comment by lbm-team** [comment]
Thank you for your feedback, this issue will be fixed in the next release

**Comment by lbm-team** [comment]
Hi @Ramin-Mardani, @MpmTwtg 

This issue is now fixed in new v4.3.0 release and mentioned in [CHANGELOG](https://github.com/Lora-net/SWL2001/blob/v4.3.0/CHANGELOG.md).

Once more thanks for your report
